packages/next/src/lib/constants.ts

```diff
@@ -1,5 +1,7 @@
 import type { ServerRuntime } from '../../types'
 
+export const NEXT_QUERY_PARAM_PREFIX = 'nextPriv'
```

Comment:
maybe a more special value with underscore so it can barely have conflicts? like `_next_param`

Reply:
It can't have special characters since it's a key in regex named parameters and query keys

Reply:
could do `nextParam` if that sounds better?

Reply:
Updated